src/SearchResults.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import FakeBookings from "./data/fakeBookings.json";
+import moment from "moment";
+
+const SearchResults = () => {
+  return (
+    <div>
```

Comment:
This is really cool. I like the way you created the table and use map to the details.

One thing, let's imagines we are going to get those FakeBookings from somewhere outside of this component. From a component above, or even load from the internet. In both cases, it would come from outside. What do you think if we receive those bookings as props? I think would make it more dynamic.